go/noise/laplace_noise.go

```diff
@@ -203,6 +236,32 @@ func laplaceLambda(l0Sensitivity int64, lInfSensitivity, epsilon float64) float6
 	return l1Sensitivity / epsilon
 }
 
+// getConfidenceIntervalLaplace computes a confidence interval that contains float64 noisedValue with probablility
+// equal to the given confidenceLevel with the given lambda.
+func getConfidenceIntervalLaplace(noisedValue float64, lambda, confidenceLevel float64) ConfidenceInterval {
+	// Finding a symmetrical confidence interval around a Laplace of (0, lambda)
+	// by computing alpha, then calculating Z_(1 - alpha/2) using inverseCDFLaplace.
+	alpha := (1 - confidenceLevel)
+	k := 1 - alpha/2
+	Zk := inverseCDFLaplace(0, lambda, k)
+	// The confidence interval will be noisedValue +/- Z_(1-alpha/2).
+	return ConfidenceInterval{noisedValue - Zk, noisedValue + Zk}
+}
+
+// inverseCDFLaplace returns Z_p where if is Y a random variable then Z_p is the value that satisfies the equation:
+// Pr( Y <= Z_p ) = p where mean and lambda are laplace distribution parameters and p is the probablity.
+func inverseCDFLaplace(mean, lambda, p float64) float64 {
```

Comment:
Calling this "inverseCDFLaplace" without mean as a parameter wouldn't make sense since it would be just a special hlper function to Confidence interval where it returns quarantines for a mean of zero. It made sense to include the mean and make it general in case of needing to use it later on.

Reply:
Resolution: Since this is a unexported function "private", It would be easy to add back mean even if we need it later on for other aggregations.